src/server/src/read-project-folders.ts

```diff
@@ -105,9 +118,13 @@ const collectResult = async (
   }
 
   const resolved = entry.fullpath()
-  const statPackageJson = await scurry.lstat(
-    `${resolved}/package.json`,
-  )
+  let statPackageJson
+  try {
+    statPackageJson = await scurry.lstat(`${resolved}/package.json`)
+  } catch {
+    // Ignore files that can't be accessed (permission denied, etc.)
+    statPackageJson = null
+  }
```

Comment:
I find this type of situation to be the perfect usage of the shorter form:

```suggestion
const statPackageJson = await scurry.lstat(
  `${resolved}/package.json`,
).catch(() => {})
```